recipes-tag/28-id-2-xpd-analysis/meta.yaml

```diff
@@ -0,0 +1,16 @@
+{% set year = "2018" %}
+{% set cycle = "1" %}
+{% set version ="0" %}
+
+package:
+  name: 28-id-2-xpd-analysis
+  version: {{ year }}-{{ cycle }}.{{ version }}
```

Comment:
We have learned that conda won't allow `-` in a version specification, so we changed `-` to `C`.